src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -276,6 +294,32 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /** initialize courses attribute by retrieving recently accessed courses from the server */
+    updateRecentlyAccessedCourses() {
+        this.courseService.findAllForDashboard().subscribe({
+            next: (res: HttpResponse<CoursesForDashboardDTO>) => {
+                if (res.body) {
+                    const { courses: courseDtos } = res.body;
+                    const courses = courseDtos.map((courseDto) => courseDto.course);
+                    this.courses = sortCourses(courses);
+                    if (this.courses.length > this.MIN_DISPLAYED_COURSES) {
+                        const lastAccessedCourseIds = this.courseAccessStorageService.getLastAccessedCourses(CourseAccessStorageService.STORAGE_KEY_DROPDOWN);
+                        this.courses = this.courses.filter((course) => lastAccessedCourseIds.includes(course.id!));
+                    }
+                    this.courses = this.courses.filter((course) => course.id !== this.courseId);
+                }
+            },
+        });
+    }
+
+    /** Navigate to a new Course */
+    switchCourse(course: Course) {
+        this.dropdownCourses = false;
```

Comment:
I guess you don't need `dropdownCourses` anymore

Reply:
👍